pkg/synthetictests/image_pulls.go

```diff
@@ -139,3 +140,16 @@ func testErrorUpdatingEndpointSlices(events monitorapi.Intervals) []*junitapi.JU
 	return newSingleEventCheckRegex(testName, errorUpdatingEndpointSlicesRegex, duplicateEventThreshold, errorUpdatingEndpointSlicesFlakeThreshold).
 		test(events.Filter(monitorapi.IsInNamespaces(sets.NewString("openshift-ovn-kubernetes"))))
 }
+
+// testInsufficientInstanceCapacity creates a junit test that allows us to count the number of times the retry logic ran
+// for the InsufficientInstanceCapacity symptom on aws.
+func testInsufficientInstanceCapacity(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+	testName := "[sig-networking] Ensure InsufficientInstanceCapacity retry logic worked on aws"
+
+	// If you see events for "reason/FailedCreate .* reconciler failed to Create machine: failed to launch instance: error
+	// creating EC2 instance: InsufficientInstanceCapacity: We currently do not have sufficient m6a.xlarge ...",
+	// it means enough was working to get that event which implies enough retries happened to allow initial openshift
+	// installation to succeed. Hence, we don't need to fail or flake this test -- so we make their thresholds as maxInt.
```

Comment:
I'm also confused, we made it through install, but machines can be requested post-install. Encountering this doesn't necessarily mean everything's ok, though I guess if not, whatever test created new Machine CRDs probably would fail itself, so this event failing is not needed.

My thought here is leaning that this is not worth adding a test for, just ignore them. Does that make sense or is there a part I'm missing?

Reply:
Ah I found it in the PR description, it will flake if encountered and then we can see how often it's happening? That does make sense.

Reply:
The fact that this event has not happened in the last 6 days makes me lean toward ignoring the event and not putting a test:

```
podman run -it corbinu/alpine-w3m -dump -cols 200 "https://search.ci.openshift.org/?search=InsufficientInstanceCapacity&maxAge=336h&context=1&type=junit&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job" | grep 'ago'|awk '{print $2" "$3" "$4" "$5}'|sort -u
junit 10 days ago
junit 13 days ago
junit 6 days ago
junit 7 days ago
junit 8 days ago
junit 9 days ago
```